settings/provisioner/distro/rhel-7.2.yml

```diff
@@ -0,0 +1,8 @@
+image:
```

Comment:
Let's discuss that with @tkammer. In the ticket I had the requirement to have distro outside virsh provisioner. The idea was to have something similar to khaleese were we have distro yaml file with all the information about specific distro. And that infroamtion were used by different modules.